packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -314,9 +281,8 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
    * @param issuer The address of issuer of the attestation
    */
   async reveal(phoneNumber: string, issuer: Address) {
-    const publicKey: string = (await this.contract.methods
-      .getDataEncryptionKey(issuer)
-      .call()) as any
+    const accounts = await this.kit.contracts.getAccounts()
+    const publicKey: string = (await accounts.getDataEncryptionKey(issuer)) as any
```

Comment:
What do we do when this is `0x0000000`?

Reply:
Great catch! Since we will no longer need to retrieve the data encryption key, I created https://github.com/celo-org/celo-monorepo/issues/1491 to remove it from here

Reply:
I don't totally follow. If we no longer need to retrieve the data encryption key, why are we doing so?

Reply:
Right now I'm doing it in here because that's technically the right change set, since this PR is not aware of the changes for #1327. Once that gets merged we can remove this here (+ the reveal itself)